### PR TITLE
#feat - added task list and task component.

### PR DESCRIPTION
**Pull request by surajmore98**
Components added:
- Task List
- Specfic Task

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+}
+
+.task-divider {
+  height: 1.5px;
+  background-color: var(--charcoal-white);
+}
+
+@media only screen and (max-width: 850px) {
+  html {
+    font-size: 12px;
```
Comment (varnita619) [inline src/App.css:219]: you should keep consistency in all units

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+function Navbar() {
+    return(
+        <div>
+            <nav className="nav-bar nav">
+                <div className="logo pl-xl">
+                    <Link to="/" className="logo-text charcoal-white">Pomodoro</Link>
+                </div>
+                {/* <ul className="nav-action ml-auto mr-md">
+                    <li className="nav-action-item">
```
Comment (varnita619) [inline src/components/Navbar.js]: remove unwanted comments

File: src/components/TaskListItem.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import { useNavigate } from "react-router-dom";
+import { useMain } from "../MainProvider";
+
+function TaskListItem({task}) {
+    const { id, title, isCompleted } = task;
+    const { setTasks, setCurrentTaskId, setFormVisible, setEdit } = useMain();
+    const navigate = useNavigate();
+
+    const taskClass = isCompleted ? 'sub-content-list-item completed' : 'sub-content-list-item';
```
Comment (varnita619) [inline src/components/TaskListItem.js:9]: good use of ternary operator

**Review by varnita619 — APPROVED** [review]
Code is clean and folder structure is good

**Review by MurtazaBagwala07 — APPROVED** [review]
folder structure looks good
code is clean
looks good to merge

suggestion : you can use barrel imports to  make imports looks more clean

**Pull request merged**